### DISCUSSION TITLE
Android // Войс в дискорде

### DISCUSSION THREAD

**Discussion by Sk7Str1p3** [hufrea/byedpi#217] [General]
Сам дс пашет но к войсу подключиться не может, какие настройки скормить надо?

**Comment by Sovenok-Hacker** [comment]
> Сам дс пашет но к войсу подключиться не может, какие настройки скормить надо?

ДС использует WebRTC, а тот UDP. Похоже, byedpi пока не поддерживает UDP.

**Comment by zimbabwe303** [comment]
> > Сам дс пашет но к войсу подключиться не может, какие настройки скормить надо?
> 
> ДС использует WebRTC, а тот UDP. Похоже, byedpi пока не поддерживает UDP.

Должен поддерживать, есть опции `--no-udp`, `--proto u` и `--udp-fake`, внутри есть функция [udp_hook](https://github.com/hufrea/byedpi/blob/bbe9522242a92b2d939a040ea2da3a4be7e9640a/extend.c#L521). Надо подбирать настройки.

**Comment by irregular-circle** [comment]
byedpi поддерживает udp, войс работает с -a1.
